OPTIMADE-Client.ipynb

```diff
@@ -44,10 +44,10 @@
     "    \"necro\",\n",
     "    \"jarvis\",\n",
     "]\n",
-    "skip_databases = {\"Materials Cloud\": [\"optimade-sample\", \"li-ion-conductors\", \"threedd\", \"sssp\"]}\n",
+    "skip_databases = {\"Materials Cloud\": [\"optimade-sample\"]}\n",
```

Comment:
currently, `li-ion-conductors` is not online (some mongodb issue), but eventually the plan is to put it online again. As i understood, it's meant as a testing database. Maybe best to keep it skipped.

Reply:
Changed.